.circleci/config.yml

```diff
@@ -1448,10 +1448,11 @@ jobs:
 
     package-bundle:
         docker:
-            - image: cimg/python:3.10
+            - image: cimg/openjdk:11.0-node
         resource_class: medium
         environment:
             RELEASE_VERSION: << pipeline.parameters.graviteeio_version >>
+            ARTIFACTORY_REPO_URL: "https://odbxikk7vo-artifactory.services.clever-cloud.com/external-dependencies-n-gravitee-all"
```

Comment:
🤔  could be a composition from existing ARTIFACTORY_URL env var + `external-dependencies-n-gravitee-all`